app/controllers/hyrax/contact_form_controller.rb

```diff
@@ -97,7 +97,7 @@ def inject_theme_views
           home_theme_view_path = Rails.root.join('app', 'views', "themes", home_page_theme.to_s)
           prepend_view_path(home_theme_view_path)
           yield
-          view_paths = original_paths
+          original_paths
```

Comment:
Change this back to `view_paths=(original_paths)`

Reply:
this is calling a method called `view_paths=()`
https://apidock.com/rails/v2.3.8/ActionController/Base/view_paths%3D/class

Reply:
Excellent, thank you for the documentation on this as well. Rubocop was throwing a spacing and parenthesis error, maybe thinking it was trying to assign a variable instead of calling the method?

Reply:
yes, rubocop is wrong on this one. You can put the disable/enable comments around this line to bypass this message.